scripts/testagent.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import json
+from Agent.readcoverage import read_coverage
+from Agent.gettest import generate_test_code
+from Agent.writedown import writedown_test_files
+def main():
+    with open('coveralls.json', 'r') as file:
+        data = json.load(file)
+
+    parser = argparse.ArgumentParser(description="用于加载API密钥。")
+    parser.add_argument('--api_key', default="4a478b99108ee30c1ae4aaa0aefe6632.X8sj7A6gaBgWh9AE", type=str, help='API密钥')

```

Comment:
Is the default key a valid key? Do I need to add variable `API_KEY` in the CI pipeline setting?